addons/medical_treatment/functions/fnc_canTreat.sqf

```diff
@@ -20,19 +20,23 @@
 
 params ["_medic", "_patient", "_bodyPart", "_classname"];
 
+private _skipForZeus = (_medic isEqualTo player && {!isNull findDisplay 312});
+
 private _config = configFile >> QGVAR(actions) >> _classname;
 
 isClass _config
 && {_patient isKindOf "CAManBase"}
-&& {_medic != _patient || {GET_NUMBER_ENTRY(_config >> "allowSelfTreatment") == 1}}
-&& {[_medic, GET_NUMBER_ENTRY(_config >> "medicRequired")] call FUNC(isMedic)}
-&& {[_medic, _patient, _config] call FUNC(canTreat_holsterCheck)}
+&& {_skipForZeus || {_medic != _patient || {GET_NUMBER_ENTRY(_config >> "allowSelfTreatment") == 1}}}
+&& {_skipForZeus || {[_medic, GET_NUMBER_ENTRY(_config >> "medicRequired")] call FUNC(isMedic)}}
+&& {_skipForZeus || {[_medic, _patient, _config] call FUNC(canTreat_holsterCheck)}}
```

Comment:
Early exit instead? This looks fugly.

Reply:
I updated it to be more efficient as well.